src/main/assets/js/court-types.ts

```diff
@@ -64,34 +64,25 @@ export class CourtTypesController {
   }
 
   //added below methods to make sure local authorities tabs is enabled and disabled when family court type is updated.
-
-
   private getAreasOfLaw(): void{
     const slug = $('#slug').val();
     $.ajax({
       url: `/courts/${slug}/local-authorities-areas-of-law`,
       method: 'get',
       success: async (res) => {
         await this.updateContent(res, this.localAuthoritiesContentId);
-        this.disableLocalAuthoritiesTab();
+        Utilities.toggleTabEnabled(this.localAuthoritiesTabId, Boolean($('#enabled').val() === 'true'));
```

Comment:
What does the Boolean(...) part do here?

Reply:
Works without it there I think! Removed it